src/createContainer.js

```diff
@@ -0,0 +1,58 @@
+/* @flow */
+
+import React from 'react';
+import Relay from 'react-relay';
+import type { RelayContainerSpec } from 'react-relay/lib/RelayContainer';
+
+function subscribe(Component) {
```

Comment:
There's probably some opportunity here to add a `subscriptions` key to the `spec` below that describes subscriptions that should be made while the component is mounted.

Seems generic enough and a nice affordance to save people from having to write `componentDidMount`/`componentDidUpdate`/`componentWillUnmount` every time.